src/Numerics/OneDimGridBase.h

```diff
@@ -38,13 +38,13 @@ struct OneDimGridBase
   typedef CT Array_t;
 
 
-  int GridTag;
-  int num_points;
-  value_type lower_bound;
-  value_type upper_bound;
+  int GridTag            = 0;
+  int num_points         = 0;
```

Comment:
Specified already in the constructor.
I feel better to consistently use the constructor.
T is always a float pointer number, so just use 0 if it is the intended value.

Reply:
Good catch.